bitnami/contour/templates/envoy/deployment.yaml

```
@@ -0,0 +1,267 @@
+{{- if .Values.envoy.enabled }}
+{{- if eq .Values.envoy.kind "Deployment" }}
+---
```

Merge both conditions (also the `---` is not required):
```suggestion
{{- if and .Values.envoy.enabled (eq .Values.envoy.kind "Deployment") }}
```